---
title: "Tumor-informed ctDNA as an objective marker for postoperative residual disease in epithelial ovarian cancer"
authors:
- Valentina Paspalj
- Christoph Grimm
- Magdalena Postl
- Christina Viktoria Tauber
- Nuria Segui
- cbrueffer
- Miguel Alcaide
- Lucia Oton
- Yilun Chen
- Lao H. Saal
- Gerda Hofstetter
- Leonhard Muellauer
- Mirjana Kessler
- Febian Trillsch

date: 2024-05-29
doi: "10.1200/JCO.2024.42.16_suppl.5544"

add_badge: true

# Schedule page publish date (NOT publication's date).
#publishDate: "2017-01-01T00:00:00Z"

# Publication type.
# Legend: 0 = Uncategorized; 1 = Conference paper; 2 = Journal article;
# 3 = Preprint / Working Paper; 4 = Report; 5 = Book; 6 = Book section;
# 7 = Thesis; 8 = Patent
publication_types: ["1"]

# Publication name and optional abbreviated publication name.
publication: "*Journal of Clinical Oncology, 2024. 42:16_suppl*"
#publication_short: ""

abstract: "**Background**<br />

The most important predictor of prognosis in patients with high-grade serous ovarian cancer (HGSOC) is complete tumor resection after primary surgery. The current practice for classifying postoperative residual disease is the subjective assessment by the physician at the end of surgery. To date, an accurate objective marker for postoperative tumor residuals is lacking. In this study we are evaluating circulating tumor DNA (ctDNA) as a predictive marker for postoperative minimum residual disease (MRD).<br />

**Methods**<br />

In this prospective multi-center feasibility study, we included 52 patients with advanced HGSOC who underwent primary debulking surgery between July 2021 and December 2023. Intraoperatively, tumor tissue from several tumor locations was evaluated. Blood samples were examined preoperatively, on day two, and day ten postoperatively. Whole genome sequencing (WGS) was used to identify structural variants (SVs), single nucleotide variants (SNVs) and indels in fresh frozen or FFPE tumor tissue to develop personalized multiplex digital polymerase chain reaction (dPCR) fingerprint assays.<br />

**Results**<br />

A high burden of SVs was detected (median 90 SVs, range 24-287) and dPCR assays successfully designed and orthogonally validated in all tumor samples. A unique dPCR fingerprint was generated for every patient, tracking one to eight biomarkers per patient (median 8). In a subset of 10 patients tissue biopsy samples from multiple tumor localizations were available. The personalized fingerprint was validated across all locations and shown to be consistent. In the overall cohort, ctDNA was positive in 96% (n=46/48) of patients preoperatively and in 87% (n=39/45) at postoperative day 10 (d10), when a plasma sample was available. Compared to patients without macroscopic residual disease, patients with postoperative tumor residuals expressed significantly higher ctDNA levels at d10 (p <0.001). Furthermore, for patients with macroscopic tumor residuals, ctDNA levels at d10 (% variant allele fraction (VAF)) remained comparable to preoperative ctDNA levels (4% decrease in median ctDNA levels). In contrast, patients with macroscopic complete resection were noted to have a 97% decrease in median ctDNA levels between pre-operative and d10 detection.<br />

**Conclusions**<br />

The present tumor-informed dPCR SV fingerprint ctDNA approach demonstrated feasibility with remarkably high detection rates pre- and postoperatively. Postoperative ctDNA levels differed substantially based on postoperative tumor residuals. These findings suggest that this personalized approach could be used to develop a dPCR SV detection assay and may have clinical utility for postoperative MRD evaluation in patients with primary advanced HGSOC.<br />"

# Summary. An optional shortened abstract.
#summary: Lorem ipsum dolor sit amet, consectetur adipiscing elit. Duis posuere tellus ac convallis placerat. Proin tincidunt magna sed ex sollicitudin condimentum.

tags:
- Cancer
- Ovarian Cancer
- Bioinformatics
- Whole Genome Sequencing
- Structural Variants
- Liquid Biopsy
- Digital PCR
- ctDNA
- Molecular Residual Disease
featured: false

links:
- name: "PDF"
  url: "https://ascopubs.org/doi/pdf/10.1200/JCO.2024.42.16_suppl.5544"
  icon: "file-pdf"
#url_pdf: ""
#url_preprint: ""
#url_code: ""
#url_dataset: ""
#url_project: ""
#url_slides: ""
#url_video: ""
#url_poster: ""
#url_source: ""


# Featured image
# To use, add an image named `featured.jpg/png` to your page's folder. 
image:
  caption: ''
  focal_point: "Center"
  preview_only: false

# Associated Projects (optional).
#   Associate this publication with one or more of your projects.
#   Simply enter your project's folder or file name without extension.
#   E.g. `internal-project` references `content/project/internal-project/index.md`.
#   Otherwise, set `projects: []`.
projects: []

# Slides (optional).
#   Associate this publication with Markdown slides.
#   Simply enter your slide deck's filename without extension.
#   E.g. `slides: "example"` references `content/slides/example/index.md`.
#   Otherwise, set `slides: ""`.
# slides: example
---

<html>
  <style>
    section {
        background: white;
        color: black;
        border-radius: 1em;
        padding: 1em;
        left: 50% }
    #inner {
        display: inline-block;
        display: flex;
        align-items: center;
        justify-content: center }
  </style>
  <section>
    <div id="inner">
      <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>
        <span style="float:left";
          class="__dimensions_badge_embed__"
          data-doi="10.1200/JCO.2024.42.16_suppl.5544"
          data-hide-zero-citations="true"
          data-legend="always">
        </span>
      <script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>
        <div style="float:right";
          data-link-target="_blank"
          data-badge-details="right"
          data-badge-type="medium-donut"
          data-doi="10.1200/JCO.2024.42.16_suppl.5544"
          data-condensed="true"
          data-hide-no-mentions="true"
          class="altmetric-embed">
        </div>
    </div>
  </section>
