---
title: "PrecisionFDA Truth Challenge V2: Calling variants from short and long reads in difficult-to-map regions"
authors:
- Nathan D. Olson
- Justin Wagner
- Jennifer McDaniel
- Sarah H. Stephens
- Samuel T. Westreich
- Anish G. Prasanna
- Elaine Johanson
- Emily Boja
- Ezekiel J. Maier
- Omar Serang
- David Jáspez
- José M. Lorenzo-Salazar
- Adrián Muñoz Barrera
- Luis A. Rubio-Rodríguez
- Carlos Flores
- Konstantinos Kyriakidis
- Andigoni Malousi
- Kishwar Shafin
- Trevor Pesout
- Miten Jain
- Benedict Paten
- Pi-Chuan Chang
- Alexey Kolesnikov
- Maria Nattestad
- Gunjan Baid
- Sidharth Goel
- Howard Yang
- Andrew Carroll
- Robert Eveleigh
- Mathieu Bourgey
- Guillaume Bourque
- Gen Li
- ChouXian Ma
- LinQi Tang
- YuanPing Du
- ShaoWei Zhang
- Jordi Morata
- Raúl Tonda
- Genís Parra
- Jean-Rémi Trotta
- cbrueffer
- Sinem Demirkaya-Budak
- Duygu Kabakci-Zorlu
- Deniz Turgut
- Özem Kalay
- Gungor Budak
- Kübra Narcı
- Elif Arslan
- Richard Brown
- Ivan J. Johnson
- Alexey Dolgoborodov
- Vladimir Semenyuk
- Amit Jain
- H. Serhat Tetikol
- Varun Jain
- Mike Ruehle
- Bryan Lajoie
- Cooper Roddey
- Severine Catreux
- Rami Mehio
- Mian Umair Ahsan
- Qian Liu
- Kai Wang
- Sayed Mohammad Ebrahim Sahraeian
- Li Tai Fang
- Marghoob Mohiyuddin
- Calvin Hung
- Chirag Jain
- Hanying Feng
- Zhipan Li
- Luoqi Chen
- Fritz J. Sedlazeck
- Justin M. Zook

date: 2022-04-27
doi: "10.1016/j.xgen.2022.100129"

add_badge: true

# Schedule page publish date (NOT publication's date).
#publishDate: "2017-01-01T00:00:00Z"

# Publication type.
# Legend: 0 = Uncategorized; 1 = Conference paper; 2 = Journal article;
# 3 = Preprint / Working Paper; 4 = Report; 5 = Book; 6 = Book section;
# 7 = Thesis; 8 = Patent
publication_types: ["2"]

# Publication name and optional abbreviated publication name.
publication: "*Cell Genomics, 2022. 2(5): 100129*"
#publication_short: ""

abstract: "The precisionFDA Truth Challenge V2 aimed to assess the state of the art of variant calling in challenging
genomic regions. Starting with FASTQs, 20 challenge participants applied their variant-calling pipelines
and submitted 64 variant call sets for one or more sequencing technologies (Illumina, PacBio HiFi, and Oxford
Nanopore Technologies). Submissions were evaluated following best practices for benchmarking small var-
iants with updated Genome in a Bottle benchmark sets and genome stratifications. Challenge submissions
included numerous innovative methods, with graph-based and machine learning methods scoring best for
short-read and long-read datasets, respectively. With machine learning approaches, combining multiple
sequencing technologies performed particularly well. Recent developments in sequencing and variant calling
have enabled benchmarking variants in challenging genomic regions, paving the way for the identification of
previously unknown clinically relevant variants."

# Summary. An optional shortened abstract.
#summary: Lorem ipsum dolor sit amet, consectetur adipiscing elit. Duis posuere tellus ac convallis placerat. Proin tincidunt magna sed ex sollicitudin condimentum.

tags:
- Genome in a Bottle
- Bioinformatics
- Machine Learning
featured: false

links:
- name: "PDF"
  url: "https://www.cell.com/action/showPdf?pii=S2666-979X%2822%2900058-1"
  icon: "file-pdf"
- name: "bioRxiv Preprint"
  url: "https://www.biorxiv.org/content/10.1101/2020.11.13.380741v4"
  icon: "biorxiv"
  icon_pack: "ai"
- name: "Project"
  url: "https://precision.fda.gov/challenges/10"
  icon: "diagram-project"
#url_pdf: ''
#url_preprint: ''
#url_code: ''
#url_dataset: ''
#url_poster: ''
#url_project: "https://precision.fda.gov/challenges/10"
#url_slides: ''
#url_source: ''
#url_video: ''

# Featured image
# To use, add an image named `featured.jpg/png` to your page's folder. 
image:
  caption: ''
  focal_point: "Center"
  preview_only: false

# Associated Projects (optional).
#   Associate this publication with one or more of your projects.
#   Simply enter your project's folder or file name without extension.
#   E.g. `internal-project` references `content/project/internal-project/index.md`.
#   Otherwise, set `projects: []`.
projects: []

# Slides (optional).
#   Associate this publication with Markdown slides.
#   Simply enter your slide deck's filename without extension.
#   E.g. `slides: "example"` references `content/slides/example/index.md`.
#   Otherwise, set `slides: ""`.
slides: example
---

<html>
  <style>
    section {
        background: white;
        color: black;
        border-radius: 1em;
        padding: 1em;
        left: 50% }
    #inner {
        display: inline-block;
        display: flex;
        align-items: center;
        justify-content: center }
  </style>
  <section>
    <div id="inner">
      <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>
        <span style="float:left";
          class="__dimensions_badge_embed__"
          data-doi="10.1016/j.xgen.2022.100129"
          data-hide-zero-citations="true"
          data-legend="always">
        </span>
      <script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>
        <div style="float:right";
          data-link-target="_blank"
          data-badge-details="right"
          data-badge-type="medium-donut"
          data-doi="10.1016/j.xgen.2022.100129"
          data-condensed="true"
          data-hide-no-mentions="true"
          class="altmetric-embed">
        </div>
    </div>
  </section>
