---
title: "The fifth international hackathon for developing computational cloud-based tools and resources for pan-structural variation and genomics"
authors:
- Sontosh K Deb
- Divya Kalra
- Jędrzej Kubica
- Erik Stricker
- Van Q. Truong
- Qiandong Zeng
- Christopher J. Fiscus
- Daniel Paiva Agustinho
- Adam Alexander
- Marlon Arciniega-Sanchez
- Lorianne Bosseau
- cbrueffer
- Astrid Canal
- Joyjit Daw
- David Enoma
- Alison Diaz-Cuevas
- Colin Diesh
- Janet M. Doolittle-Hall
- Luis Fernandez-Luna
- Tina Han
- Wolfram Höps
- Peiming Peter Huang
- Tony Huang
- Michal Bogumil Izydorczyk
- Farhang Jaryani
- Rupesh K. Kesharwani
- Shaheerah Khan
- Sina Majidian
- Ayan Malakar
- Tania Girão Mangolini
- Sejal Modha
- Mauricio Moldes
- Rajarshi Mondal
- Abdullah Al Nahid
- Chi-Lam Poon
- Sagayamary Sagayaradj
- Philippe Sanio
- Tania Sepulveda-Morales
- Muhammad Shahzaib
- Muhammad Sohail Raza
- Trinh Tat
- Ishaan Thota
- Umran Yaman
- Jason Yeung
- Qiyi Yu
- Xinchang Zheng
- Medhat Mahmoud
- Fritz J. Sedlazeck
- Ben Busby

date: 2024-06-27
doi: "10.12688/f1000research.148237.1"

add_badge: true

# Schedule page publish date (NOT publication's date).
#publishDate: "2017-01-01T00:00:00Z"

# Publication type.
# Legend: 0 = Uncategorized; 1 = Conference paper; 2 = Journal article;
# 3 = Preprint / Working Paper; 4 = Report; 5 = Book; 6 = Book section;
# 7 = Thesis; 8 = Patent
publication_types: ["2"]

# Publication name and optional abbreviated publication name.
publication: "*F1000Research, 2024. 13:708*"
#publication_short: ""

abstract: "**Background**<br />

The goal of the Fifth Annual Baylor College of Medicine & DNAnexus Structural Variation Hackathon was to push forward the research on structural variants (SVs) by rapidly developing and deploying open-source software. The event took place in-person and virtually in August 2023, when 49 scientists from 14 countries and 8 U.S. states collaboratively worked on projects to address critical gaps in the field of genomics. The hackathon projects concentrated on developing bioinformatic workflows for the following challenges: RNA transcriptome comparison, simulation of mosaic variations, metagenomics, Mendelian variation, SVs in plant genomics, and assembly vs. mapping SV calling comparisons.<br />

**Methods**<br />

As a starting point we used publicly available data from state-of-the-art long- and short-read sequencing technologies. The workflows developed during the hackathon incorporated open-source software, as well as scripts written using Bash and Python. Moreover, we leveraged the advantages of Docker and Snakemake for workflow automation.<br />

**Results**<br />

The results of the hackathon consists of six prototype bioinformatic workflows that use open-source software for SV research. We made the workflows scalable and modular for usability and reproducibility. Furthermore, we tested the workflows on example public data to show that the workflows can work. The code and the data produced during the event have been made publicly available on GitHub (https://github.com/collaborativebioinformatics) to reproduce and built upon in the future.<br />

**Conclusions**<br />

The following sections describe the motivation, lessons learned, and software produced by teams during the hackathon. Here, we describe in detail the objectives, value propositions, implementation, and use cases for our workflows. In summary, the article reports the advancements in the development of software for SV detection made during the hackathon."

# Summary. An optional shortened abstract.
#summary: Lorem ipsum dolor sit amet, consectetur adipiscing elit. Duis posuere tellus ac convallis placerat. Proin tincidunt magna sed ex sollicitudin condimentum.

tags:
- Bioinformatics
- RNA Sequencing
- Structural Variants
- Metagenomics
- Long-read sequencing
- Hackathon
featured: false

links:
- name: "PDF"
  url: "https://f1000research.com/articles/13-708/v1/pdf?article_uuid=cfd27152-5f69-4e48-8057-6f5787e0407d"
  icon: "file-pdf"
- name: "Code"
  url: "https://github.com/collaborativebioinformatics"
  icon: "code"
#url_pdf: ""
#url_preprint: ""
#url_code: ""
#url_dataset: ""
#url_project: ""
#url_slides: ""
#url_video: ""
#url_poster: ""
#url_source: ""


# Featured image
# To use, add an image named `featured.jpg/png` to your page's folder. 
image:
  caption: ''
  focal_point: "Center"
  preview_only: false

# Associated Projects (optional).
#   Associate this publication with one or more of your projects.
#   Simply enter your project's folder or file name without extension.
#   E.g. `internal-project` references `content/project/internal-project/index.md`.
#   Otherwise, set `projects: []`.
projects: []

# Slides (optional).
#   Associate this publication with Markdown slides.
#   Simply enter your slide deck's filename without extension.
#   E.g. `slides: "example"` references `content/slides/example/index.md`.
#   Otherwise, set `slides: ""`.
# slides: example
---

<html>
  <style>
    section {
        background: white;
        color: black;
        border-radius: 1em;
        padding: 1em;
        left: 50% }
    #inner {
        display: inline-block;
        display: flex;
        align-items: center;
        justify-content: center }
  </style>
  <section>
    <div id="inner">
      <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>
        <span style="float:left";
          class="__dimensions_badge_embed__"
          data-doi="10.12688/f1000research.148237.1"
          data-hide-zero-citations="true"
          data-legend="always">
        </span>
      <script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>
        <div style="float:right";
          data-link-target="_blank"
          data-badge-details="right"
          data-badge-type="medium-donut"
          data-doi="10.12688/f1000research.148237.1"
          data-condensed="true"
          data-hide-no-mentions="true"
          class="altmetric-embed">
        </div>
    </div>
  </section>
