<!doctype html><html lang=en-us><head><meta charset=utf-8><meta name=viewport content="width=device-width,initial-scale=1"><meta http-equiv=x-ua-compatible content="IE=edge"><meta name=generator content="Wowchemy 5.7.0 for Hugo"><link rel=stylesheet href=/css/vendor-bundle.min.16f785cdb553c8c4431db6775122af35.css media=print onload='this.media="all"'><link rel=stylesheet href=https://cdn.jsdelivr.net/npm/academicons@1.9.2/css/academicons.min.css integrity="sha512-KlJCpRsLf+KKu2VQa5vmRuClRFjxc5lXO03ixZt82HZUk41+1I0bD8KBSA0fY290ayMfWYI9udIqeOWSu1/uZg==" crossorigin=anonymous media=print onload='this.media="all"'><link rel=stylesheet href=https://cdn.jsdelivr.net/npm/leaflet@1.7.1/dist/leaflet.min.css integrity crossorigin=anonymous media=print onload='this.media="all"'><link rel=stylesheet href=/css/wowchemy.5928d6f6b5eee121dc6141d2cdd0af46.css><link rel=stylesheet href=/css/libs/chroma/github-light.min.css title=hl-light media=print onload='this.media="all"'><link rel=stylesheet href=/css/libs/chroma/dracula.min.css title=hl-dark media=print onload='this.media="all"' disabled><meta name=author content="Christian Brueffer"><meta name=description content="**Introduction/Background**<br />
Complete tumor resection is the most relevant prognostic factor for overall survival in high grade serous ovarian cancer (HGSOC) patients. The current standard for classification of postoperative residual disease is surgeon's subjective evaluation at the end of surgery. Thus, a reliable objective predictive marker is currently missing.<br />
**Methodology**<br />
In this prospective single-center study, patients with advanced HGSOC (≥ FIGO IIIA1), who underwent surgery between July 2021 and December 2022, were included. Tumor tissue from multiple intraperitoneal locations was obtained intraoperatively and blood samples were collected preoperatively, at day 2 and 10 postoperatively. Low-coverage whole genome sequencing (WGS) was used to identify structural variants (SV), single nucleotide variants (SNVs) and insertion deletions (InDels) in tumor tissue in order to develop personalized digital PCR (dPCR) fingerprint assays.<br />
**Results**<br />
In all tumor samples of the 31 included patients, dPCR assays were successfully developed and validated, with a median of 5 biomarkers (SVs and SNVs) per patient. For each patient, an individual SV profile could be established, which remained largely constant throughout multiple tumor localizations of each patient. 30/31 (97%) patients had circulating tumor DNA (ctDNA) detected at baseline before surgery at levels ranging from 0.0005% to 31% variant allele frequency. ctDNA was persistently detected in all patients with macroscopic tumor residuals. A significant decrease in ctDNA was observed in 15/20 (75%) patients with advanced HGSOC and in 6/6 (100%) patients with stage IIIA1-IIIB disease, who had macroscopic complete resection. In 8/20 (40%) patients with complete resection, ctDNA decreased below the detection limit.<br />
**Conclusion**<br />
In this feasibility study, tumor-informed ctDNA was preoperatively detectable in 97% participants. In patients with multiple tumor biopsies, the fingerprint was consistent for all tumor locations. A decrease in ctDNA detection correlated with complete tumor resection.<br />
**Disclosures**<br />
Study partially funded by SAGA Diagnostics."><link rel=alternate hreflang=en-us href=https://www.brueffer.io/publication/2023_esgo/><link rel=canonical href=https://www.brueffer.io/publication/2023_esgo/><link rel=manifest href=/manifest.webmanifest><link rel=icon type=image/png href=/media/icon_hu7422bdd1ad1d5b8840d6088d6ba7ed4c_15338_32x32_fill_lanczos_center_3.png><link rel=apple-touch-icon type=image/png href=/media/icon_hu7422bdd1ad1d5b8840d6088d6ba7ed4c_15338_180x180_fill_lanczos_center_3.png><meta name=theme-color content="#1565c0"><meta property="twitter:card" content="summary_large_image"><meta property="twitter:site" content="@cbrueffer"><meta property="twitter:creator" content="@cbrueffer"><meta property="twitter:image" content="https://www.brueffer.io/publication/2023_esgo/featured.png"><meta property="og:site_name" content="Christian Brueffer"><meta property="og:url" content="https://www.brueffer.io/publication/2023_esgo/"><meta property="og:title" content="#898 Tumor-informed ctDNA detection as a marker for postoperative residual disease in epithelial ovarian cancer – results of a feasibility study | Christian Brueffer"><meta property="og:description" content="**Introduction/Background**<br />
Complete tumor resection is the most relevant prognostic factor for overall survival in high grade serous ovarian cancer (HGSOC) patients. The current standard for classification of postoperative residual disease is surgeon's subjective evaluation at the end of surgery. Thus, a reliable objective predictive marker is currently missing.<br />
**Methodology**<br />
In this prospective single-center study, patients with advanced HGSOC (≥ FIGO IIIA1), who underwent surgery between July 2021 and December 2022, were included. Tumor tissue from multiple intraperitoneal locations was obtained intraoperatively and blood samples were collected preoperatively, at day 2 and 10 postoperatively. Low-coverage whole genome sequencing (WGS) was used to identify structural variants (SV), single nucleotide variants (SNVs) and insertion deletions (InDels) in tumor tissue in order to develop personalized digital PCR (dPCR) fingerprint assays.<br />
**Results**<br />
In all tumor samples of the 31 included patients, dPCR assays were successfully developed and validated, with a median of 5 biomarkers (SVs and SNVs) per patient. For each patient, an individual SV profile could be established, which remained largely constant throughout multiple tumor localizations of each patient. 30/31 (97%) patients had circulating tumor DNA (ctDNA) detected at baseline before surgery at levels ranging from 0.0005% to 31% variant allele frequency. ctDNA was persistently detected in all patients with macroscopic tumor residuals. A significant decrease in ctDNA was observed in 15/20 (75%) patients with advanced HGSOC and in 6/6 (100%) patients with stage IIIA1-IIIB disease, who had macroscopic complete resection. In 8/20 (40%) patients with complete resection, ctDNA decreased below the detection limit.<br />
**Conclusion**<br />
In this feasibility study, tumor-informed ctDNA was preoperatively detectable in 97% participants. In patients with multiple tumor biopsies, the fingerprint was consistent for all tumor locations. A decrease in ctDNA detection correlated with complete tumor resection.<br />
**Disclosures**<br />
Study partially funded by SAGA Diagnostics."><meta property="og:image" content="https://www.brueffer.io/publication/2023_esgo/featured.png"><meta property="og:locale" content="en-us"><meta property="article:published_time" content="2023-09-27T00:00:00+00:00"><meta property="article:modified_time" content="2023-09-27T00:00:00+00:00"><script type=application/ld+json>{"@context":"https://schema.org","@type":"Article","mainEntityOfPage":{"@type":"WebPage","@id":"https://www.brueffer.io/publication/2023_esgo/"},"headline":"#898 Tumor-informed ctDNA detection as a marker for postoperative residual disease in epithelial ovarian cancer – results of a feasibility study","image":["https://www.brueffer.io/publication/2023_esgo/featured.png"],"datePublished":"2023-09-27T00:00:00Z","dateModified":"2023-09-27T00:00:00Z","author":{"@type":"Person","name":"Magdalena Postl"},"publisher":{"@type":"Organization","name":"Christian Brueffer","logo":{"@type":"ImageObject","url":"https://www.brueffer.io/media/icon_hu7422bdd1ad1d5b8840d6088d6ba7ed4c_15338_192x192_fill_lanczos_center_3.png"}},"description":"**Introduction/Background**\u003cbr /\u003e\nComplete tumor resection is the most relevant prognostic factor for overall survival in high grade serous ovarian cancer (HGSOC) patients. The current standard for classification of postoperative residual disease is surgeon's subjective evaluation at the end of surgery. Thus, a reliable objective predictive marker is currently missing.\u003cbr /\u003e\n**Methodology**\u003cbr /\u003e\nIn this prospective single-center study, patients with advanced HGSOC (≥ FIGO IIIA1), who underwent surgery between July 2021 and December 2022, were included. Tumor tissue from multiple intraperitoneal locations was obtained intraoperatively and blood samples were collected preoperatively, at day 2 and 10 postoperatively. Low-coverage whole genome sequencing (WGS) was used to identify structural variants (SV), single nucleotide variants (SNVs) and insertion deletions (InDels) in tumor tissue in order to develop personalized digital PCR (dPCR) fingerprint assays.\u003cbr /\u003e\n**Results**\u003cbr /\u003e\nIn all tumor samples of the 31 included patients, dPCR assays were successfully developed and validated, with a median of 5 biomarkers (SVs and SNVs) per patient. For each patient, an individual SV profile could be established, which remained largely constant throughout multiple tumor localizations of each patient. 30/31 (97%) patients had circulating tumor DNA (ctDNA) detected at baseline before surgery at levels ranging from 0.0005% to 31% variant allele frequency. ctDNA was persistently detected in all patients with macroscopic tumor residuals. A significant decrease in ctDNA was observed in 15/20 (75%) patients with advanced HGSOC and in 6/6 (100%) patients with stage IIIA1-IIIB disease, who had macroscopic complete resection. In 8/20 (40%) patients with complete resection, ctDNA decreased below the detection limit.\u003cbr /\u003e\n**Conclusion**\u003cbr /\u003e\nIn this feasibility study, tumor-informed ctDNA was preoperatively detectable in 97% participants. In patients with multiple tumor biopsies, the fingerprint was consistent for all tumor locations. A decrease in ctDNA detection correlated with complete tumor resection.\u003cbr /\u003e\n**Disclosures**\u003cbr /\u003e\nStudy partially funded by SAGA Diagnostics."}</script><title>#898 Tumor-informed ctDNA detection as a marker for postoperative residual disease in epithelial ovarian cancer – results of a feasibility study | Christian Brueffer</title></head><body id=top data-spy=scroll data-offset=70 data-target=#TableOfContents class=page-wrapper data-wc-page-id=e7a6e592057b69187f3bc72dcc31925f><script src=/js/wowchemy-init.min.ec9d49ca50e4b80bdb08f0417a28ed84.js></script><aside class=search-modal id=search><div class=container><section class=search-header><div class="row no-gutters justify-content-between mb-3"><div class=col-6><h1>Search</h1></div><div class="col-6 col-search-close"><a class=js-search href=# aria-label=Close><i class="fas fa-times-circle text-muted" aria-hidden=true></i></a></div></div><div id=search-box><input name=q id=search-query placeholder=Search... autocapitalize=off autocomplete=off autocorrect=off spellcheck=false type=search class=form-control aria-label=Search...></div></section><section class=section-search-results><div id=search-hits></div></section></div></aside><div class="page-header header--fixed"><header><nav class="navbar navbar-expand-lg navbar-light compensate-for-scrollbar" id=navbar-main><div class=container-xl><div class="d-none d-lg-inline-flex"><a class=navbar-brand href=/>Christian Brueffer</a></div><button type=button class=navbar-toggler data-toggle=collapse data-target=#navbar-content aria-controls=navbar-content aria-expanded=false aria-label="Toggle navigation">
<span><i class="fas fa-bars"></i></span></button><div class="navbar-brand-mobile-wrapper d-inline-flex d-lg-none"><a class=navbar-brand href=/>Christian Brueffer</a></div><div class="navbar-collapse main-menu-item collapse justify-content-start" id=navbar-content><ul class="navbar-nav d-md-inline-flex"><li class=nav-item><a class=nav-link href=/><span>About Me</span></a></li><li class=nav-item><a class=nav-link href=/#consulting><span>Consulting</span></a></li><li class=nav-item><a class=nav-link href=/#posts><span>Posts</span></a></li><li class=nav-item><a class=nav-link href=/#projects><span>Projects</span></a></li><li class=nav-item><a class=nav-link href=/event/><span>Talks</span></a></li><li class="nav-item dropdown"><a href=# class="nav-link dropdown-toggle" data-toggle=dropdown aria-haspopup=true><span>Research</span><span class=caret></span></a><div class=dropdown-menu><a class=dropdown-item href=/#tags><span>Popular Topics</span></a>
<a class=dropdown-item href=/#featured><span>Featured Publications</span></a>
<a class=dropdown-item href=/#publications><span>Recent Publications</span></a>
<a class=dropdown-item href=/publication/><span>All Publications</span></a></div></li><li class=nav-item><a class=nav-link href=/#contact><span>Contact</span></a></li></ul></div><ul class="nav-icons navbar-nav flex-row ml-auto d-flex pl-md-2"><li class=nav-item><a class="nav-link js-search" href=# aria-label=Search><i class="fas fa-search" aria-hidden=true></i></a></li><li class="nav-item dropdown theme-dropdown"><a href=# class=nav-link data-toggle=dropdown aria-haspopup=true aria-label="Display preferences"><i class="fas fa-moon" aria-hidden=true></i></a><div class=dropdown-menu><a href=# class="dropdown-item js-set-theme-light"><span>Light</span></a>
<a href=# class="dropdown-item js-set-theme-dark"><span>Dark</span></a>
<a href=# class="dropdown-item js-set-theme-auto"><span>Automatic</span></a></div></li></ul></div></nav></header></div><div class=page-body><div class=pub><div class="article-container pt-3"><h1>#898 Tumor-informed ctDNA detection as a marker for postoperative residual disease in epithelial ovarian cancer – results of a feasibility study</h1><div class=article-metadata><div><span>Magdalena Postl</span>, <span>Valentina Paspalj</span>, <span class=author-highlighted>Christian Brueffer</span>, <span>Nuria Segui Gracia</span>, <span>Miguel Alcaide</span>, <span>Lucia Oton</span>, <span>Yilun Chen</span>, <span>Lao H. Saal</span>, <span>Gerda Hofstetter</span>, <span>Leonhard Müllauer</span>, <span>Stephan Polterauer</span>, <span>Christoph Grimm</span></div><span class=article-date>September, 2023</span></div><div class="btn-links mb-3"><a class="btn btn-outline-primary btn-page-header" href=https://doi.org/10.1136/ijgc-2023-ESGO.60 target=_blank rel=noopener>DOI</a>
<a class="btn btn-outline-primary btn-page-header" href=https://ijgc.bmj.com/content/ijgc/33/Suppl_3/A38.2.full.pdf target=_blank rel=noopener><i class="fas fa-file-pdf mr-1"></i>PDF</a></div></div><div class="article-header article-container featured-image-wrapper mt-4 mb-4" style=max-width:330px;max-height:440px><div style=position:relative><img src=/publication/2023_esgo/featured_huc09277338ecf48ca24defb5dbb1718f0_31606_720x2500_fit_q75_h2_lanczos_3.webp width=330 height=440 alt class=featured-image></div></div><div class=article-container><h3>Abstract</h3><p class=pub-abstract><strong>Introduction/Background</strong><br>Complete tumor resection is the most relevant prognostic factor for overall survival in high grade serous ovarian cancer (HGSOC) patients. The current standard for classification of postoperative residual disease is surgeon&rsquo;s subjective evaluation at the end of surgery. Thus, a reliable objective predictive marker is currently missing.<br><strong>Methodology</strong><br>In this prospective single-center study, patients with advanced HGSOC (≥ FIGO IIIA1), who underwent surgery between July 2021 and December 2022, were included. Tumor tissue from multiple intraperitoneal locations was obtained intraoperatively and blood samples were collected preoperatively, at day 2 and 10 postoperatively. Low-coverage whole genome sequencing (WGS) was used to identify structural variants (SV), single nucleotide variants (SNVs) and insertion deletions (InDels) in tumor tissue in order to develop personalized digital PCR (dPCR) fingerprint assays.<br><strong>Results</strong><br>In all tumor samples of the 31 included patients, dPCR assays were successfully developed and validated, with a median of 5 biomarkers (SVs and SNVs) per patient. For each patient, an individual SV profile could be established, which remained largely constant throughout multiple tumor localizations of each patient. 30/31 (97%) patients had circulating tumor DNA (ctDNA) detected at baseline before surgery at levels ranging from 0.0005% to 31% variant allele frequency. ctDNA was persistently detected in all patients with macroscopic tumor residuals. A significant decrease in ctDNA was observed in 15/20 (75%) patients with advanced HGSOC and in 6/6 (100%) patients with stage IIIA1-IIIB disease, who had macroscopic complete resection. In 8/20 (40%) patients with complete resection, ctDNA decreased below the detection limit.<br><strong>Conclusion</strong><br>In this feasibility study, tumor-informed ctDNA was preoperatively detectable in 97% participants. In patients with multiple tumor biopsies, the fingerprint was consistent for all tumor locations. A decrease in ctDNA detection correlated with complete tumor resection.<br><strong>Disclosures</strong><br>Study partially funded by SAGA Diagnostics.</p><div class=row><div class=col-md-1></div><div class=col-md-10><div class=row><div class="col-12 col-md-3 pub-row-heading">Type</div><div class="col-12 col-md-9"><a href=/publication/#1>Conference paper</a></div></div></div><div class=col-md-1></div></div><div class="d-md-none space-below"></div><div class=row><div class=col-md-1></div><div class=col-md-10><div class=row><div class="col-12 col-md-3 pub-row-heading">Publication</div><div class="col-12 col-md-9"><em>International Journal of Gynecologic Cancer, 2023. 33:A38-A39</em></div></div></div><div class=col-md-1></div></div><div class="d-md-none space-below"></div><div class=space-below></div><div class=article-style><html><style>section{background:#fff;color:#000;border-radius:1em;padding:1em;left:50%}#inner{display:inline-block;display:flex;align-items:center;justify-content:center}</style><section><div id=inner><script type=text/javascript src=https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js></script>
<span style=float:left ; class=__dimensions_badge_embed__ data-doi=10.1136/ijgc-2023-ESGO.60 data-hide-zero-citations=true data-legend=always></span><script async src=https://badge.dimensions.ai/badge.js></script><div style=float:right ; data-link-target=_blank data-badge-details=right data-badge-type=medium-donut data-doi=10.1136/ijgc-2023-ESGO.60 data-condensed=true data-hide-no-mentions=true class=altmetric-embed></div></div></section></div><div class=article-tags><a class="badge badge-light" href=/tag/cancer/>Cancer</a>
<a class="badge badge-light" href=/tag/ovarian-cancer/>Ovarian Cancer</a>
<a class="badge badge-light" href=/tag/bioinformatics/>Bioinformatics</a>
<a class="badge badge-light" href=/tag/whole-genome-sequencing/>Whole Genome Sequencing</a>
<a class="badge badge-light" href=/tag/structural-variants/>Structural Variants</a>
<a class="badge badge-light" href=/tag/liquid-biopsy/>Liquid Biopsy</a>
<a class="badge badge-light" href=/tag/digital-pcr/>Digital PCR</a>
<a class="badge badge-light" href=/tag/ctdna/>ctDNA</a></div><div class=share-box><ul class=share><li><a href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.brueffer.io%2Fpublication%2F2023_esgo%2F&text=%23898%E2%80%85Tumor-informed+ctDNA+detection+as+a+marker+for+postoperative+residual+disease+in+epithelial+ovarian+cancer+%E2%80%93+results+of+a+feasibility+study" target=_blank rel=noopener class=share-btn-twitter aria-label=twitter><i class="fab fa-twitter"></i></a></li><li><a href="https://www.facebook.com/sharer.php?u=https%3A%2F%2Fwww.brueffer.io%2Fpublication%2F2023_esgo%2F&t=%23898%E2%80%85Tumor-informed+ctDNA+detection+as+a+marker+for+postoperative+residual+disease+in+epithelial+ovarian+cancer+%E2%80%93+results+of+a+feasibility+study" target=_blank rel=noopener class=share-btn-facebook aria-label=facebook><i class="fab fa-facebook"></i></a></li><li><a href="mailto:?subject=%23898%E2%80%85Tumor-informed%20ctDNA%20detection%20as%20a%20marker%20for%20postoperative%20residual%20disease%20in%20epithelial%20ovarian%20cancer%20%E2%80%93%20results%20of%20a%20feasibility%20study&body=https%3A%2F%2Fwww.brueffer.io%2Fpublication%2F2023_esgo%2F" target=_blank rel=noopener class=share-btn-email aria-label=envelope><i class="fas fa-envelope"></i></a></li><li><a href="https://www.linkedin.com/shareArticle?url=https%3A%2F%2Fwww.brueffer.io%2Fpublication%2F2023_esgo%2F&title=%23898%E2%80%85Tumor-informed+ctDNA+detection+as+a+marker+for+postoperative+residual+disease+in+epithelial+ovarian+cancer+%E2%80%93+results+of+a+feasibility+study" target=_blank rel=noopener class=share-btn-linkedin aria-label=linkedin-in><i class="fab fa-linkedin-in"></i></a></li><li><a href="https://web.whatsapp.com/send?text=%23898%E2%80%85Tumor-informed+ctDNA+detection+as+a+marker+for+postoperative+residual+disease+in+epithelial+ovarian+cancer+%E2%80%93+results+of+a+feasibility+study%20https%3A%2F%2Fwww.brueffer.io%2Fpublication%2F2023_esgo%2F" target=_blank rel=noopener class=share-btn-whatsapp aria-label=whatsapp><i class="fab fa-whatsapp"></i></a></li><li><a href="https://service.weibo.com/share/share.php?url=https%3A%2F%2Fwww.brueffer.io%2Fpublication%2F2023_esgo%2F&title=%23898%E2%80%85Tumor-informed+ctDNA+detection+as+a+marker+for+postoperative+residual+disease+in+epithelial+ovarian+cancer+%E2%80%93+results+of+a+feasibility+study" target=_blank rel=noopener class=share-btn-weibo aria-label=weibo><i class="fab fa-weibo"></i></a></li></ul></div><div class="media author-card content-widget-hr"><a href=https://www.brueffer.io><img class="avatar mr-3 avatar-circle" src=/authors/cbrueffer/avatar_hu17f80c4ec327e7baf25f17a8159016b3_537581_270x270_fill_lanczos_center_3.png alt="Christian Brueffer"></a><div class=media-body><h5 class=card-title><a href=https://www.brueffer.io>Christian Brueffer</a></h5><h6 class=card-subtitle>Bioinformatician and Data Scientist</h6><p class=card-text>Freelance Bioinformatician and Data Scientist with interests including disease biology and diagnostics, particularly in cancer, and open source bioinformatics.</p><ul class=network-icon aria-hidden=true><li><a href=mailto:christian@brueffer.io><i class="fas fa-envelope"></i></a></li><li><a href=https://twitter.com/cbrueffer target=_blank rel=noopener><i class="fab fa-twitter"></i></a></li><li><a href=https://mastodon.social/@cbrueffer target=_blank rel=noopener><i class="fab fa-mastodon"></i></a></li><li><a href=https://www.linkedin.com/in/cbrueffer target=_blank rel=noopener><i class="fab fa-linkedin"></i></a></li><li><a href="https://scholar.google.com/citations?user=BFnR7W8AAAAJ" target=_blank rel=noopener><i class="ai ai-google-scholar"></i></a></li><li><a href=https://www.researchgate.net/profile/Christian_Brueffer2 target=_blank rel=noopener><i class="ai ai-researchgate"></i></a></li><li><a href=https://orcid.org/0000-0002-3826-0989 target=_blank rel=noopener><i class="ai ai-orcid"></i></a></li><li><a href=https://github.com/cbrueffer target=_blank rel=noopener><i class="fab fa-github"></i></a></li></ul></div></div></div></div></div><div class=page-footer><div class=container><footer class=site-footer><p class="powered-by copyright-license-text">© 2023 Christian Brueffer. This work is licensed under <a href=https://creativecommons.org/licenses/by-nc-nd/4.0 rel="noopener noreferrer" target=_blank>CC BY NC ND 4.0</a></p><p class="powered-by footer-license-icons"><a href=https://creativecommons.org/licenses/by-nc-nd/4.0 rel="noopener noreferrer" target=_blank aria-label="Creative Commons"><i class="fab fa-creative-commons fa-2x" aria-hidden=true></i>
<i class="fab fa-creative-commons-by fa-2x" aria-hidden=true></i>
<i class="fab fa-creative-commons-nc fa-2x" aria-hidden=true></i>
<i class="fab fa-creative-commons-nd fa-2x" aria-hidden=true></i></a></p><p class=powered-by>Published with <a href="https://wowchemy.com/?utm_campaign=poweredby" target=_blank rel=noopener>Wowchemy</a> — the free, <a href=https://github.com/wowchemy/wowchemy-hugo-themes target=_blank rel=noopener>open source</a> website builder that empowers creators.</p></footer></div></div><script src=/js/vendor-bundle.min.46271ef31da3f018e9cd1b59300aa265.js></script>
<script src=https://cdn.jsdelivr.net/npm/leaflet@1.7.1/dist/leaflet.min.js integrity crossorigin=anonymous></script>
<script id=search-hit-fuse-template type=text/x-template>
    <div class="search-hit" id="summary-{{key}}">
      <div class="search-hit-content">
        <div class="search-hit-name">
          <a href="{{relpermalink}}">{{title}}</a>
          <div class="article-metadata search-hit-type">{{type}}</div>
          <p class="search-hit-description">{{snippet}}</p>
        </div>
      </div>
    </div>
  </script><script src=https://cdn.jsdelivr.net/gh/krisk/Fuse@v3.2.1/dist/fuse.min.js integrity="sha512-o38bmzBGX+hD3JHWUFCDA09btWaqrNmoJ3RXLlrysA7PP01Kgs4UlE4MhelE1v5dJR3+cxlR4qQlotsW7jKsnw==" crossorigin=anonymous></script>
<script src=https://cdn.jsdelivr.net/gh/julmot/mark.js@8.11.1/dist/jquery.mark.min.js integrity="sha512-mhbv5DqBMgrWL+32MmsDOt/OAvqr/cHimk6B8y/bx/xS88MVkYGPiVv2ixKVrkywF2qHplNRUvFsAHUdxZ3Krg==" crossorigin=anonymous></script>
<script id=page-data type=application/json>{"use_headroom":true}</script><script src=/js/wowchemy-headroom.db4755770454eb63685f8de785c0a172.js type=module></script>
<script src=/en/js/wowchemy.min.8621d706896ab6d11dd8fd7042e20931.js></script>
<script src=/js/wowchemy-map.a26e9d2f7238ba5b868384f1c5bc6477.js type=module></script><div id=modal class="modal fade" role=dialog><div class=modal-dialog><div class=modal-content><div class=modal-header><h5 class=modal-title>Cite</h5><button type=button class=close data-dismiss=modal aria-label=Close>
<span aria-hidden=true>&#215;</span></button></div><div class=modal-body><pre><code></code></pre></div><div class=modal-footer><a class="btn btn-outline-primary my-1 js-copy-cite" href=# target=_blank><i class="fas fa-copy"></i> Copy</a>
<a class="btn btn-outline-primary my-1 js-download-cite" href=# target=_blank><i class="fas fa-download"></i> Download</a><div id=modal-error></div></div></div></div></div><script src=/js/wowchemy-publication.68f8d7090562ca65fc6d3cb3f8f2d2cb.js type=module></script>
<script type=text/javascript src=https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js></script>
<script async src=https://badge.dimensions.ai/badge.js></script></body></html>