---
title: "NeoCircle: Investigating circulating tumor DNA dynamics as a predictor of survival in primary breast cancer"
authors:
- Anthony M. George
- Yilun Chen
- Sergii Gladchuk
- Miguel Alcaide
- Hina Dalal
- Pei Meng
- cbrueffer
- Hani Saghir
- Siker Kimbung
- Lucia Oton
- Ida Skarping
- Daniel Förnvik
- Gabriella Honeth
- Karen Howarth
- Åke Borg
- Anna Ehinger
- Martin Malmberg
- Lisa Rydén
- Niklas Loman
- Lao H. Saal

date: 2025-05-28
doi: "10.1200/JCO.2025.43.16_suppl.3057"

add_badge: true

# Schedule page publish date (NOT publication's date).
#publishDate: "2017-01-01T00:00:00Z"

# Publication type.
# Legend: 0 = Uncategorized; 1 = Conference paper; 2 = Journal article;
# 3 = Preprint / Working Paper; 4 = Report; 5 = Book; 6 = Book section;
# 7 = Thesis; 8 = Patent
publication_types: ["1"]

# Publication name and optional abbreviated publication name.
publication: "*Journal of Clinical Oncology, 2025. 43:16_suppl*"
#publication_short: ""

abstract: "**Background**<br />

Persistent circulating tumor DNA (ctDNA) detection during neoadjuvant treatment (NAT) of early-breast cancer (EBC) indicates high-risk disease. Following surgical resection, ctDNA-positivity indicates molecular residual disease (MRD) and heralds occult metastatic disease relapse. To incorporate ctDNA into EBC management, scalable and widely accessible diagnostic methods are necessary. Here we apply an ultrasensitive, personalized tumor-informed approach to ctDNA analysis leveraging structural variant (SV) detection using a novel multiplex digital PCR (dPCR) technology.<br />

**Methods**<br />

116 patients with stage I-III EBC (31.0% TNBC, 43.1% HR+/HER2- and 24.1% HER2+) and eligible for NAT were recruited through the prospective SCAN-B study (NCT02306096, substudy NeoCircle) between December 2014 and March 2019 and have been analyzed for ctDNA. Whole genome sequencing was performed on tumor material and personalized multiplex dPCR assays tracking up to 16 SVs were used for ctDNA monitoring. Plasma samples were collected at baseline, during NAT, pre- and post-surgery and at 6-monthly intervals during follow up.<br />

**Results**<br />

High baseline detection was observed across all stages and subtypes (90.5% overall), and ctDNA-positivity at end-of-NAT (end-NAT) was a significant predictor of eventual disease relapse and death (relapse-free interval, RFI, hazard ratio, HR, 3.7, 95% CI 1.4-9.7; overall survival HR 7.7, 95% CI 2.2-26.6). A significant association was observed between end-NAT ctDNA clearance and pathological complete response (pCR), whereas non-pCR by itself was not a significant predictor of relapse or death in this cohort. At one or more post-operative timepoints, MRD+ was detected in 10 patients who experienced distant recurrence, with lead times up to 4 years (median 13.9 months, range 1.8-47.7 months). Similarly, ctDNA was detected in 3 of 4 patients with local recurrences and 1 of 2 patients with CNS-only recurrences. For 2 patients without presentation of clinical recurrence to date, ctDNA was detected post-operatively, with subsequent clearance during follow-up. Post-operative MRD associated with poor RFI (HR 45.5, 95% CI 13.0-159.8) and OS (HR 15.3, 95% CI 4.5-52.9).<br />

**Conclusions**<br />

In this analysis of 116 patients from a prospective study in patients with EBC receiving NAT, we monitored ctDNA using an ultrasensitive tumor-informed dPCR assay tracking patient-specific SVs. ctDNA detection post-NAT and prior to surgery was associated with high-risk of disease relapse and death, outperforming pCR. Moreover, post-operative ctDNA detection was also significantly associated with disease relapse and death, with long lead-times over standard-of-care clinical assessments. These findings further validate the feasibility of SVs as an MRD analyte and support the clinical use of this approach in EBC."

# Summary. An optional shortened abstract.
#summary: Lorem ipsum dolor sit amet, consectetur adipiscing elit. Duis posuere tellus ac convallis placerat. Proin tincidunt magna sed ex sollicitudin condimentum.

tags:
- Cancer
- Breast Cancer
- Bioinformatics
- Whole Genome Sequencing
- Structural Variants
- Liquid Biopsy
- Digital PCR
- ctDNA
- Molecular Residual Disease
featured: false

links:
- name: "PDF"
  url: "https://ascopubs.org/doi/pdf/10.1200/JCO.2025.43.16_suppl.3057"
  icon: "file-pdf"
#url_pdf: ""
#url_preprint: ""
#url_code: ""
#url_dataset: ""
#url_project: ""
#url_slides: ""
#url_video: ""
#url_poster: ""
#url_source: ""


# Featured image
# To use, add an image named `featured.jpg/png` to your page's folder. 
image:
  caption: ''
  focal_point: "Center"
  preview_only: false

# Associated Projects (optional).
#   Associate this publication with one or more of your projects.
#   Simply enter your project's folder or file name without extension.
#   E.g. `internal-project` references `content/project/internal-project/index.md`.
#   Otherwise, set `projects: []`.
projects: []

# Slides (optional).
#   Associate this publication with Markdown slides.
#   Simply enter your slide deck's filename without extension.
#   E.g. `slides: "example"` references `content/slides/example/index.md`.
#   Otherwise, set `slides: ""`.
# slides: example
---

<html>
  <style>
    section {
        background: white;
        color: black;
        border-radius: 1em;
        padding: 1em;
        left: 50% }
    #inner {
        display: inline-block;
        display: flex;
        align-items: center;
        justify-content: center }
  </style>
  <section>
    <div id="inner">
      <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>
        <span style="float:left";
          class="__dimensions_badge_embed__"
          data-doi="10.1200/JCO.2025.43.16_suppl.3057"
          data-hide-zero-citations="true"
          data-legend="always">
        </span>
      <script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>
        <div style="float:right";
          data-link-target="_blank"
          data-badge-details="right"
          data-badge-type="medium-donut"
          data-doi="10.1200/JCO.2025.43.16_suppl.3057"
          data-condensed="true"
          data-hide-no-mentions="true"
          class="altmetric-embed">
        </div>
    </div>
  </section>
