---
title: "Defining the mutational landscape of 3,217 primary breast cancer transcriptomes through large-scale RNA-seq within the Sweden Cancerome Analysis Network: Breast Project (SCAN-B; NCT03430492)."
authors:
- cbrueffer
- Sergii Gladchuk
- Christof Winter
- Johan Vallon-Christersson
- Cecilia Hegardt
- Jari Häkkinen
- Anthony M George
- Yilun Chen
- Anna Ehinger
- Christer Larsson
- Niklas Loman
- Martin Malmberg
- Lisa Rydén
- Åke Borg
- Lao H. Saal

date: 2020-05-25
doi: "10.1200/JCO.2020.38.15_suppl.518"

add_badge: true

# Schedule page publish date (NOT publication's date).
#publishDate: "2017-01-01T00:00:00Z"

# Publication type.
# Legend: 0 = Uncategorized; 1 = Conference paper; 2 = Journal article;
# 3 = Preprint / Working Paper; 4 = Report; 5 = Book; 6 = Book section;
# 7 = Thesis; 8 = Patent
publication_types: ["1"]

# Publication name and optional abbreviated publication name.
publication: "*Journal of Clinical Oncology, 2020. 38:15_suppl*"
#publication_short: ""

abstract: "**Background**<br />

Breast cancer is a disease of genomic alterations, of which the complete panorama of somatic mutations and how these relate to molecular subtypes and therapy response is incompletely understood. The Sweden Cancerome Analysis Network-Breast project (SCAN-B; ClinicalTrials.gov NCT02306096) is a multi-center population-based ongoing prospective observational study elucidating the global transcriptomic profiles for thousands of patients and tumors using RNA sequencing. Since September 2010, over 15,000 patients with breast cancer have been enrolled at 9 hospitals across a wide geography of Sweden, comprising greater than 90% of all eligible patients in the catchment area.<br />

**Methods**<br />

Within SCAN-B, we developed an optimized bioinformatics pipeline for detection of single nucleotide variants and small insertions and deletions from RNA-seq data. From this, we describe the mutational landscape of 3,217 primary breast cancer transcriptomes, and relate it to patient overall survival in a real-world setting (median follow-up 75 months, range 2-105 months).<br />

**Results**<br />

We demonstrate that RNA-seq can be used to call mutations in important breast cancer genes such as *PIK3CA*, *TP53*, *ESR1*, and *ERBB2*, as well as mutation status of key molecular pathways and tumor mutational burden, identify mutations in one or more potentially druggable genes in 85.3% percent of cases, and reveal significant relationships to patient outcome within specific treatment groups. To make this rich and growing mutational portraiture of breast cancer available for the wider research community, we developed an open source interactive web application, SCAN-B MutationExplorer, publicly accessible at http://oncogenomics.bmc.lu.se/MutationExplorer.<br />

**Conclusions**<br />
These results add another dimension to the use of RNA-seq as a potential clinical tool, where both gene expression-based signatures and gene mutation-based biomarkers can be interrogated simultaneously and in real-time within one week of tumor sampling."

# Summary. An optional shortened abstract.
#summary: Lorem ipsum dolor sit amet, consectetur adipiscing elit. Duis posuere tellus ac convallis placerat. Proin tincidunt magna sed ex sollicitudin condimentum.

tags:
- Breast Cancer
- SCAN-B
- RNA Sequencing
- Open Source Software
- PhD
- Bioinformatics
featured: false

links:
#- name: "PDF"
#  url: ""
#  icon: "file-pdf"
#url_pdf: ""
#url_preprint: ""
#url_code: ""
#url_dataset: ""
#url_project: ""
#url_slides: ""
#url_video: ""
#url_poster: ""
#url_source: ""


# Featured image
# To use, add an image named `featured.jpg/png` to your page's folder. 
image:
  caption: ''
  focal_point: "Center"
  preview_only: false

# Associated Projects (optional).
#   Associate this publication with one or more of your projects.
#   Simply enter your project's folder or file name without extension.
#   E.g. `internal-project` references `content/project/internal-project/index.md`.
#   Otherwise, set `projects: []`.
projects: []

# Slides (optional).
#   Associate this publication with Markdown slides.
#   Simply enter your slide deck's filename without extension.
#   E.g. `slides: "example"` references `content/slides/example/index.md`.
#   Otherwise, set `slides: ""`.
# slides: example
---

<html>
  <style>
    section {
        background: white;
        color: black;
        border-radius: 1em;
        padding: 1em;
        left: 50% }
    #inner {
        display: inline-block;
        display: flex;
        align-items: center;
        justify-content: center }
  </style>
  <section>
    <div id="inner">
      <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>
        <span style="float:left";
          class="__dimensions_badge_embed__"
          data-doi="10.1200/JCO.2020.38.15_suppl.518"
          data-hide-zero-citations="true"
          data-legend="always">
        </span>
      <script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>
        <div style="float:right";
          data-link-target="_blank"
          data-badge-details="right"
          data-badge-type="medium-donut"
          data-doi="10.1200/JCO.2020.38.15_suppl.518"
          data-condensed="true"
          data-hide-no-mentions="true"
          class="altmetric-embed">
        </div>
    </div>
  </section>
