---
title: "#898 Tumor-informed ctDNA detection as a marker for postoperative residual disease in epithelial ovarian cancer – results of a feasibility study"
authors:
- Magdalena Postl
- Valentina Paspalj
- cbrueffer
- Nuria Segui Gracia
- Miguel Alcaide
- Lucia Oton
- Yilun Chen
- Lao H. Saal
- Gerda Hofstetter
- Leonhard Müllauer
- Stephan Polterauer
- Christoph Grimm

date: 2023-09-27
doi: "10.1136/ijgc-2023-ESGO.60"

add_badge: true

# Schedule page publish date (NOT publication's date).
#publishDate: "2017-01-01T00:00:00Z"

# Publication type.
# Legend: 0 = Uncategorized; 1 = Conference paper; 2 = Journal article;
# 3 = Preprint / Working Paper; 4 = Report; 5 = Book; 6 = Book section;
# 7 = Thesis; 8 = Patent
publication_types: ["1"]

# Publication name and optional abbreviated publication name.
publication: "*International Journal of Gynecologic Cancer, 2023. 33:A38-A39*"
#publication_short: ""

abstract: "**Introduction/Background**<br />

Complete tumor resection is the most relevant prognostic factor for overall survival in high grade serous ovarian cancer (HGSOC) patients. The current standard for classification of postoperative residual disease is surgeon's subjective evaluation at the end of surgery. Thus, a reliable objective predictive marker is currently missing.<br />

**Methodology**<br />

In this prospective single-center study, patients with advanced HGSOC (≥ FIGO IIIA1), who underwent surgery between July 2021 and December 2022, were included. Tumor tissue from multiple intraperitoneal locations was obtained intraoperatively and blood samples were collected preoperatively, at day 2 and 10 postoperatively. Low-coverage whole genome sequencing (WGS) was used to identify structural variants (SV), single nucleotide variants (SNVs) and insertion deletions (InDels) in tumor tissue in order to develop personalized digital PCR (dPCR) fingerprint assays.<br />

**Results**<br />

In all tumor samples of the 31 included patients, dPCR assays were successfully developed and validated, with a median of 5 biomarkers (SVs and SNVs) per patient. For each patient, an individual SV profile could be established, which remained largely constant throughout multiple tumor localizations of each patient. 30/31 (97%) patients had circulating tumor DNA (ctDNA) detected at baseline before surgery at levels ranging from 0.0005% to 31% variant allele frequency. ctDNA was persistently detected in all patients with macroscopic tumor residuals. A significant decrease in ctDNA was observed in 15/20 (75%) patients with advanced HGSOC and in 6/6 (100%) patients with stage IIIA1-IIIB disease, who had macroscopic complete resection. In 8/20 (40%) patients with complete resection, ctDNA decreased below the detection limit.<br />

**Conclusion**<br />

In this feasibility study, tumor-informed ctDNA was preoperatively detectable in 97% participants. In patients with multiple tumor biopsies, the fingerprint was consistent for all tumor locations. A decrease in ctDNA detection correlated with complete tumor resection.<br />

**Disclosures**<br />

Study partially funded by SAGA Diagnostics."

# Summary. An optional shortened abstract.
#summary: Lorem ipsum dolor sit amet, consectetur adipiscing elit. Duis posuere tellus ac convallis placerat. Proin tincidunt magna sed ex sollicitudin condimentum.

tags:
- Cancer
- Ovarian Cancer
- Bioinformatics
- Whole Genome Sequencing
- Structural Variants
- Liquid Biopsy
- Digital PCR
featured: false

links:
- name: "PDF"
  url: "https://ijgc.bmj.com/content/ijgc/33/Suppl_3/A38.2.full.pdf"
  icon: "file-pdf"
#url_pdf: ""
#url_preprint: ""
#url_code: ""
#url_dataset: ""
#url_project: ""
#url_slides: ""
#url_video: ""
#url_poster: ""
#url_source: ""


# Featured image
# To use, add an image named `featured.jpg/png` to your page's folder. 
image:
  caption: ''
  focal_point: "Center"
  preview_only: false

# Associated Projects (optional).
#   Associate this publication with one or more of your projects.
#   Simply enter your project's folder or file name without extension.
#   E.g. `internal-project` references `content/project/internal-project/index.md`.
#   Otherwise, set `projects: []`.
projects: []

# Slides (optional).
#   Associate this publication with Markdown slides.
#   Simply enter your slide deck's filename without extension.
#   E.g. `slides: "example"` references `content/slides/example/index.md`.
#   Otherwise, set `slides: ""`.
# slides: example
---

<html>
  <style>
    section {
        background: white;
        color: black;
        border-radius: 1em;
        padding: 1em;
        left: 50% }
    #inner {
        display: inline-block;
        display: flex;
        align-items: center;
        justify-content: center }
  </style>
  <section>
    <div id="inner">
      <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>
        <span style="float:left";
          class="__dimensions_badge_embed__"
          data-doi="10.1136/ijgc-2023-ESGO.60"
          data-hide-zero-citations="true"
          data-legend="always">
        </span>
      <script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>
        <div style="float:right";
          data-link-target="_blank"
          data-badge-details="right"
          data-badge-type="medium-donut"
          data-doi="10.1136/ijgc-2023-ESGO.60"
          data-condensed="true"
          data-hide-no-mentions="true"
          class="altmetric-embed">
        </div>
    </div>
  </section>
