<!doctype html><html lang=en-us><head><meta charset=utf-8><meta name=viewport content="width=device-width,initial-scale=1"><meta http-equiv=x-ua-compatible content="IE=edge"><meta name=generator content="Wowchemy 5.7.0 for Hugo"><link rel=stylesheet href=/css/vendor-bundle.min.16f785cdb553c8c4431db6775122af35.css media=print onload='this.media="all"'><link rel=stylesheet href=https://cdn.jsdelivr.net/npm/academicons@1.9.2/css/academicons.min.css integrity="sha512-KlJCpRsLf+KKu2VQa5vmRuClRFjxc5lXO03ixZt82HZUk41+1I0bD8KBSA0fY290ayMfWYI9udIqeOWSu1/uZg==" crossorigin=anonymous media=print onload='this.media="all"'><link rel=stylesheet href=https://cdn.jsdelivr.net/npm/leaflet@1.7.1/dist/leaflet.min.css integrity crossorigin=anonymous media=print onload='this.media="all"'><link rel=stylesheet href=/css/wowchemy.5928d6f6b5eee121dc6141d2cdd0af46.css><link rel=stylesheet href=/css/libs/chroma/github-light.min.css title=hl-light media=print onload='this.media="all"'><link rel=stylesheet href=/css/libs/chroma/dracula.min.css title=hl-dark media=print onload='this.media="all"' disabled><meta name=author content="Christian Brueffer"><meta name=description content="**Purpose**<br /> In early breast cancer (BC), five conventional biomarkers—estrogen receptor (ER), progesterone receptor (PgR), human epidermal growth factor receptor 2 (HER2), Ki67, and Nottingham histologic grade (NHG)—are used to determine prognosis and treatment. We aimed to develop classifiers for these biomarkers that were based on tumor mRNA sequencing (RNA-seq), compare classification performance, and test whether such predictors could add value for risk stratification.<br /><br />
**Methods**<br /> In total, 3,678 patients with BC were studied. For 405 tumors, a comprehensive multi-rater histopathologic evaluation was performed. Using RNA-seq data, single-gene classifiers and multigene classifiers (MGCs) were trained on consensus histopathology labels. Trained classifiers were tested on a prospective population-based series of 3,273 BCs that included a median follow-up of 52 months (Sweden Cancerome Analysis Network—Breast [SCAN-B], ClinicalTrials.gov identifier: NCT02306096), and results were evaluated by agreement statistics and Kaplan-Meier and Cox survival analyses.<br /><br />
**Results**<br /> Pathologist concordance was high for ER, PgR, and HER2 (average κ, 0.920, 0.891, and 0.899, respectively) but moderate for Ki67 and NHG (average κ, 0.734 and 0.581). Concordance between RNA-seq classifiers and histopathology for the independent cohort of 3,273 was similar to interpathologist concordance. Patients with discordant classifications, predicted as hormone responsive by histopathology but non–hormone responsive by MGC, had significantly inferior overall survival compared with patients who had concordant results. This extended to patients who received no adjuvant therapy (hazard ratio [HR], 3.19; 95% CI, 1.19 to 8.57), or endocrine therapy alone (HR, 2.64; 95% CI, 1.55 to 4.51). For cases identified as hormone responsive by histopathology and who received endocrine therapy alone, the MGC hormone-responsive classifier remained significant after multivariable adjustment (HR, 2.45; 95% CI, 1.39 to 4.34).<br /><br />
**Conclusion**<br /> Classification error rates for RNA-seq–based classifiers for the five key BC biomarkers generally were equivalent to conventional histopathology. However, RNA-seq classifiers provided added clinical value in particular for tumors determined by histopathology to be hormone responsive but by RNA-seq to be hormone insensitive."><link rel=alternate hreflang=en-us href=https://www.brueffer.io/publication/2018_scan-b_biomarker_prediction/><link rel=canonical href=https://www.brueffer.io/publication/2018_scan-b_biomarker_prediction/><link rel=manifest href=/manifest.webmanifest><link rel=icon type=image/png href=/media/icon_hu7422bdd1ad1d5b8840d6088d6ba7ed4c_15338_32x32_fill_lanczos_center_3.png><link rel=apple-touch-icon type=image/png href=/media/icon_hu7422bdd1ad1d5b8840d6088d6ba7ed4c_15338_180x180_fill_lanczos_center_3.png><meta name=theme-color content="#1565c0"><meta property="twitter:card" content="summary_large_image"><meta property="twitter:site" content="@cbrueffer"><meta property="twitter:creator" content="@cbrueffer"><meta property="twitter:image" content="https://www.brueffer.io/publication/2018_scan-b_biomarker_prediction/featured.png"><meta property="og:site_name" content="Christian Brueffer"><meta property="og:url" content="https://www.brueffer.io/publication/2018_scan-b_biomarker_prediction/"><meta property="og:title" content="Clinical Value of RNA Sequencing-Based Classifiers for Prediction of the Five Conventional Breast Cancer Biomarkers: A Report From the Population-Based Multicenter Sweden Cancerome Analysis Network–Breast Initiative | Christian Brueffer"><meta property="og:description" content="**Purpose**<br /> In early breast cancer (BC), five conventional biomarkers—estrogen receptor (ER), progesterone receptor (PgR), human epidermal growth factor receptor 2 (HER2), Ki67, and Nottingham histologic grade (NHG)—are used to determine prognosis and treatment. We aimed to develop classifiers for these biomarkers that were based on tumor mRNA sequencing (RNA-seq), compare classification performance, and test whether such predictors could add value for risk stratification.<br /><br />
**Methods**<br /> In total, 3,678 patients with BC were studied. For 405 tumors, a comprehensive multi-rater histopathologic evaluation was performed. Using RNA-seq data, single-gene classifiers and multigene classifiers (MGCs) were trained on consensus histopathology labels. Trained classifiers were tested on a prospective population-based series of 3,273 BCs that included a median follow-up of 52 months (Sweden Cancerome Analysis Network—Breast [SCAN-B], ClinicalTrials.gov identifier: NCT02306096), and results were evaluated by agreement statistics and Kaplan-Meier and Cox survival analyses.<br /><br />
**Results**<br /> Pathologist concordance was high for ER, PgR, and HER2 (average κ, 0.920, 0.891, and 0.899, respectively) but moderate for Ki67 and NHG (average κ, 0.734 and 0.581). Concordance between RNA-seq classifiers and histopathology for the independent cohort of 3,273 was similar to interpathologist concordance. Patients with discordant classifications, predicted as hormone responsive by histopathology but non–hormone responsive by MGC, had significantly inferior overall survival compared with patients who had concordant results. This extended to patients who received no adjuvant therapy (hazard ratio [HR], 3.19; 95% CI, 1.19 to 8.57), or endocrine therapy alone (HR, 2.64; 95% CI, 1.55 to 4.51). For cases identified as hormone responsive by histopathology and who received endocrine therapy alone, the MGC hormone-responsive classifier remained significant after multivariable adjustment (HR, 2.45; 95% CI, 1.39 to 4.34).<br /><br />
**Conclusion**<br /> Classification error rates for RNA-seq–based classifiers for the five key BC biomarkers generally were equivalent to conventional histopathology. However, RNA-seq classifiers provided added clinical value in particular for tumors determined by histopathology to be hormone responsive but by RNA-seq to be hormone insensitive."><meta property="og:image" content="https://www.brueffer.io/publication/2018_scan-b_biomarker_prediction/featured.png"><meta property="og:locale" content="en-us"><meta property="article:published_time" content="2018-03-01T00:00:00+00:00"><meta property="article:modified_time" content="2018-03-01T00:00:00+00:00"><script type=application/ld+json>{"@context":"https://schema.org","@type":"Article","mainEntityOfPage":{"@type":"WebPage","@id":"https://www.brueffer.io/publication/2018_scan-b_biomarker_prediction/"},"headline":"Clinical Value of RNA Sequencing-Based Classifiers for Prediction of the Five Conventional Breast Cancer Biomarkers: A Report From the Population-Based Multicenter Sweden Cancerome Analysis Network–Breast Initiative","image":["https://www.brueffer.io/publication/2018_scan-b_biomarker_prediction/featured.png"],"datePublished":"2018-03-01T00:00:00Z","dateModified":"2018-03-01T00:00:00Z","author":{"@type":"Person","name":"Christian Brueffer"},"publisher":{"@type":"Organization","name":"Christian Brueffer","logo":{"@type":"ImageObject","url":"https://www.brueffer.io/media/icon_hu7422bdd1ad1d5b8840d6088d6ba7ed4c_15338_192x192_fill_lanczos_center_3.png"}},"description":"**Purpose**\u003cbr /\u003e In early breast cancer (BC), five conventional biomarkers—estrogen receptor (ER), progesterone receptor (PgR), human epidermal growth factor receptor 2 (HER2), Ki67, and Nottingham histologic grade (NHG)—are used to determine prognosis and treatment. We aimed to develop classifiers for these biomarkers that were based on tumor mRNA sequencing (RNA-seq), compare classification performance, and test whether such predictors could add value for risk stratification.\u003cbr /\u003e\u003cbr /\u003e\n**Methods**\u003cbr /\u003e In total, 3,678 patients with BC were studied. For 405 tumors, a comprehensive multi-rater histopathologic evaluation was performed. Using RNA-seq data, single-gene classifiers and multigene classifiers (MGCs) were trained on consensus histopathology labels. Trained classifiers were tested on a prospective population-based series of 3,273 BCs that included a median follow-up of 52 months (Sweden Cancerome Analysis Network—Breast [SCAN-B], ClinicalTrials.gov identifier: NCT02306096), and results were evaluated by agreement statistics and Kaplan-Meier and Cox survival analyses.\u003cbr /\u003e\u003cbr /\u003e\n**Results**\u003cbr /\u003e Pathologist concordance was high for ER, PgR, and HER2 (average κ, 0.920, 0.891, and 0.899, respectively) but moderate for Ki67 and NHG (average κ, 0.734 and 0.581). Concordance between RNA-seq classifiers and histopathology for the independent cohort of 3,273 was similar to interpathologist concordance. Patients with discordant classifications, predicted as hormone responsive by histopathology but non–hormone responsive by MGC, had significantly inferior overall survival compared with patients who had concordant results. This extended to patients who received no adjuvant therapy (hazard ratio [HR], 3.19; 95% CI, 1.19 to 8.57), or endocrine therapy alone (HR, 2.64; 95% CI, 1.55 to 4.51). For cases identified as hormone responsive by histopathology and who received endocrine therapy alone, the MGC hormone-responsive classifier remained significant after multivariable adjustment (HR, 2.45; 95% CI, 1.39 to 4.34).\u003cbr /\u003e\u003cbr /\u003e\n**Conclusion**\u003cbr /\u003e Classification error rates for RNA-seq–based classifiers for the five key BC biomarkers generally were equivalent to conventional histopathology. However, RNA-seq classifiers provided added clinical value in particular for tumors determined by histopathology to be hormone responsive but by RNA-seq to be hormone insensitive."}</script><title>Clinical Value of RNA Sequencing-Based Classifiers for Prediction of the Five Conventional Breast Cancer Biomarkers: A Report From the Population-Based Multicenter Sweden Cancerome Analysis Network–Breast Initiative | Christian Brueffer</title></head><body id=top data-spy=scroll data-offset=70 data-target=#TableOfContents class=page-wrapper data-wc-page-id=818f53f20f4d8c4bae88d8db4df65dd4><script src=/js/wowchemy-init.min.ec9d49ca50e4b80bdb08f0417a28ed84.js></script><aside class=search-modal id=search><div class=container><section class=search-header><div class="row no-gutters justify-content-between mb-3"><div class=col-6><h1>Search</h1></div><div class="col-6 col-search-close"><a class=js-search href=# aria-label=Close><i class="fas fa-times-circle text-muted" aria-hidden=true></i></a></div></div><div id=search-box><input name=q id=search-query placeholder=Search... autocapitalize=off autocomplete=off autocorrect=off spellcheck=false type=search class=form-control aria-label=Search...></div></section><section class=section-search-results><div id=search-hits></div></section></div></aside><div class="page-header header--fixed"><header><nav class="navbar navbar-expand-lg navbar-light compensate-for-scrollbar" id=navbar-main><div class=container-xl><div class="d-none d-lg-inline-flex"><a class=navbar-brand href=/>Christian Brueffer</a></div><button type=button class=navbar-toggler data-toggle=collapse data-target=#navbar-content aria-controls=navbar-content aria-expanded=false aria-label="Toggle navigation">
<span><i class="fas fa-bars"></i></span></button><div class="navbar-brand-mobile-wrapper d-inline-flex d-lg-none"><a class=navbar-brand href=/>Christian Brueffer</a></div><div class="navbar-collapse main-menu-item collapse justify-content-start" id=navbar-content><ul class="navbar-nav d-md-inline-flex"><li class=nav-item><a class=nav-link href=/><span>About Me</span></a></li><li class=nav-item><a class=nav-link href=/#consulting><span>Consulting</span></a></li><li class=nav-item><a class=nav-link href=/#posts><span>Posts</span></a></li><li class=nav-item><a class=nav-link href=/#projects><span>Projects</span></a></li><li class=nav-item><a class=nav-link href=/event/><span>Talks</span></a></li><li class="nav-item dropdown"><a href=# class="nav-link dropdown-toggle" data-toggle=dropdown aria-haspopup=true><span>Research</span><span class=caret></span></a><div class=dropdown-menu><a class=dropdown-item href=/#tags><span>Popular Topics</span></a>
<a class=dropdown-item href=/#featured><span>Featured Publications</span></a>
<a class=dropdown-item href=/#publications><span>Recent Publications</span></a>
<a class=dropdown-item href=/publication/><span>All Publications</span></a></div></li><li class=nav-item><a class=nav-link href=/#contact><span>Contact</span></a></li></ul></div><ul class="nav-icons navbar-nav flex-row ml-auto d-flex pl-md-2"><li class=nav-item><a class="nav-link js-search" href=# aria-label=Search><i class="fas fa-search" aria-hidden=true></i></a></li><li class="nav-item dropdown theme-dropdown"><a href=# class=nav-link data-toggle=dropdown aria-haspopup=true aria-label="Display preferences"><i class="fas fa-moon" aria-hidden=true></i></a><div class=dropdown-menu><a href=# class="dropdown-item js-set-theme-light"><span>Light</span></a>
<a href=# class="dropdown-item js-set-theme-dark"><span>Dark</span></a>
<a href=# class="dropdown-item js-set-theme-auto"><span>Automatic</span></a></div></li></ul></div></nav></header></div><div class=page-body><div class=pub><div class="article-container pt-3"><h1>Clinical Value of RNA Sequencing-Based Classifiers for Prediction of the Five Conventional Breast Cancer Biomarkers: A Report From the Population-Based Multicenter Sweden Cancerome Analysis Network–Breast Initiative</h1><div class=article-metadata><div><span class=author-highlighted>Christian Brueffer</span><i class="author-notes fas fa-info-circle" data-toggle=tooltip title="Equal contribution"></i>, <span>Johan Vallon-Christersson</span><i class="author-notes fas fa-info-circle" data-toggle=tooltip title="Equal contribution"></i>, <span>Dorthe Grabau</span>, <span>Anna Ehinger</span>, <span>Jari Häkkinen</span>, <span>Cecilia Hegardt</span>, <span>Janne Malina</span>, <span>Yilun Chen</span>, <span>Pär-Ola Bendahl</span>, <span>Jonas Manjer</span>, <span>Martin Malmberg</span>, <span>Christer Larsson</span>, <span>Niklas Loman</span>, <span>Lisa Rydén</span>, <span>Åke Borg</span>, <span>Lao H. Saal</span></div><span class=article-date>March, 2018</span></div><div class="btn-links mb-3"><a class="btn btn-outline-primary btn-page-header" href=https://doi.org/10.1200/PO.17.00135 target=_blank rel=noopener>DOI</a>
<a class="btn btn-outline-primary btn-page-header" href=https://ascopubs.org/doi/pdfdirect/10.1200/PO.17.00135 target=_blank rel=noopener><i class="fas fa-file-pdf mr-1"></i>PDF</a>
<a class="btn btn-outline-primary btn-page-header" href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE81540" target=_blank rel=noopener><i class="fas fa-database mr-1"></i>Dataset</a>
<a class="btn btn-outline-primary btn-page-header" href=/project/phd><i class="fas fa-graduation-cap mr-1"></i>PhD Project</a></div></div><div class="article-header article-container featured-image-wrapper mt-4 mb-4" style=max-width:651px;max-height:878px><div style=position:relative><img src=/publication/2018_scan-b_biomarker_prediction/featured_hu97d17c13c2e92d62a3fa4e1636e0a1ee_231550_720x2500_fit_q75_h2_lanczos_3.webp width=651 height=878 alt class=featured-image></div></div><div class=article-container><h3>Abstract</h3><p class=pub-abstract><strong>Purpose</strong><br>In early breast cancer (BC), five conventional biomarkers—estrogen receptor (ER), progesterone receptor (PgR), human epidermal growth factor receptor 2 (HER2), Ki67, and Nottingham histologic grade (NHG)—are used to determine prognosis and treatment. We aimed to develop classifiers for these biomarkers that were based on tumor mRNA sequencing (RNA-seq), compare classification performance, and test whether such predictors could add value for risk stratification.<br><br><strong>Methods</strong><br>In total, 3,678 patients with BC were studied. For 405 tumors, a comprehensive multi-rater histopathologic evaluation was performed. Using RNA-seq data, single-gene classifiers and multigene classifiers (MGCs) were trained on consensus histopathology labels. Trained classifiers were tested on a prospective population-based series of 3,273 BCs that included a median follow-up of 52 months (Sweden Cancerome Analysis Network—Breast [SCAN-B], ClinicalTrials.gov identifier: NCT02306096), and results were evaluated by agreement statistics and Kaplan-Meier and Cox survival analyses.<br><br><strong>Results</strong><br>Pathologist concordance was high for ER, PgR, and HER2 (average κ, 0.920, 0.891, and 0.899, respectively) but moderate for Ki67 and NHG (average κ, 0.734 and 0.581). Concordance between RNA-seq classifiers and histopathology for the independent cohort of 3,273 was similar to interpathologist concordance. Patients with discordant classifications, predicted as hormone responsive by histopathology but non–hormone responsive by MGC, had significantly inferior overall survival compared with patients who had concordant results. This extended to patients who received no adjuvant therapy (hazard ratio [HR], 3.19; 95% CI, 1.19 to 8.57), or endocrine therapy alone (HR, 2.64; 95% CI, 1.55 to 4.51). For cases identified as hormone responsive by histopathology and who received endocrine therapy alone, the MGC hormone-responsive classifier remained significant after multivariable adjustment (HR, 2.45; 95% CI, 1.39 to 4.34).<br><br><strong>Conclusion</strong><br>Classification error rates for RNA-seq–based classifiers for the five key BC biomarkers generally were equivalent to conventional histopathology. However, RNA-seq classifiers provided added clinical value in particular for tumors determined by histopathology to be hormone responsive but by RNA-seq to be hormone insensitive.</p><div class=row><div class=col-md-1></div><div class=col-md-10><div class=row><div class="col-12 col-md-3 pub-row-heading">Type</div><div class="col-12 col-md-9"><a href=/publication/#2>Journal article</a></div></div></div><div class=col-md-1></div></div><div class="d-md-none space-below"></div><div class=row><div class=col-md-1></div><div class=col-md-10><div class=row><div class="col-12 col-md-3 pub-row-heading">Publication</div><div class="col-12 col-md-9"><em>JCO Precision Oncology, 2018. 2:1&ndash;18</em></div></div></div><div class=col-md-1></div></div><div class="d-md-none space-below"></div><div class=space-below></div><div class=article-style><html><style>section{background:#fff;color:#000;border-radius:1em;padding:1em;left:50%}#inner{display:inline-block;display:flex;align-items:center;justify-content:center}</style><section><div id=inner><script type=text/javascript src=https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js></script>
<span style=float:left ; class=__dimensions_badge_embed__ data-doi=10.1200/PO.17.00135 data-hide-zero-citations=true data-legend=always></span><script async src=https://badge.dimensions.ai/badge.js></script><div style=float:right ; data-link-target=_blank data-badge-details=right data-badge-type=medium-donut data-doi=10.1200/PO.17.00135 data-condensed=true data-hide-no-mentions=true class=altmetric-embed></div></div></section></div><div class=article-tags><a class="badge badge-light" href=/tag/cancer/>Cancer</a>
<a class="badge badge-light" href=/tag/breast-cancer/>Breast Cancer</a>
<a class="badge badge-light" href=/tag/scan-b/>SCAN-B</a>
<a class="badge badge-light" href=/tag/rna-sequencing/>RNA Sequencing</a>
<a class="badge badge-light" href=/tag/machine-learning/>Machine Learning</a>
<a class="badge badge-light" href=/tag/phd/>PhD</a>
<a class="badge badge-light" href=/tag/bioinformatics/>Bioinformatics</a></div><div class=share-box><ul class=share><li><a href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.brueffer.io%2Fpublication%2F2018_scan-b_biomarker_prediction%2F&text=Clinical+Value+of+RNA+Sequencing-Based+Classifiers+for+Prediction+of+the+Five+Conventional+Breast+Cancer+Biomarkers%3A+A+Report+From+the+Population-Based+Multicenter+Sweden+Cancerome+Analysis+Network%E2%80%93Breast+Initiative" target=_blank rel=noopener class=share-btn-twitter aria-label=twitter><i class="fab fa-twitter"></i></a></li><li><a href="https://www.facebook.com/sharer.php?u=https%3A%2F%2Fwww.brueffer.io%2Fpublication%2F2018_scan-b_biomarker_prediction%2F&t=Clinical+Value+of+RNA+Sequencing-Based+Classifiers+for+Prediction+of+the+Five+Conventional+Breast+Cancer+Biomarkers%3A+A+Report+From+the+Population-Based+Multicenter+Sweden+Cancerome+Analysis+Network%E2%80%93Breast+Initiative" target=_blank rel=noopener class=share-btn-facebook aria-label=facebook><i class="fab fa-facebook"></i></a></li><li><a href="mailto:?subject=Clinical%20Value%20of%20RNA%20Sequencing-Based%20Classifiers%20for%20Prediction%20of%20the%20Five%20Conventional%20Breast%20Cancer%20Biomarkers%3A%20A%20Report%20From%20the%20Population-Based%20Multicenter%20Sweden%20Cancerome%20Analysis%20Network%E2%80%93Breast%20Initiative&body=https%3A%2F%2Fwww.brueffer.io%2Fpublication%2F2018_scan-b_biomarker_prediction%2F" target=_blank rel=noopener class=share-btn-email aria-label=envelope><i class="fas fa-envelope"></i></a></li><li><a href="https://www.linkedin.com/shareArticle?url=https%3A%2F%2Fwww.brueffer.io%2Fpublication%2F2018_scan-b_biomarker_prediction%2F&title=Clinical+Value+of+RNA+Sequencing-Based+Classifiers+for+Prediction+of+the+Five+Conventional+Breast+Cancer+Biomarkers%3A+A+Report+From+the+Population-Based+Multicenter+Sweden+Cancerome+Analysis+Network%E2%80%93Breast+Initiative" target=_blank rel=noopener class=share-btn-linkedin aria-label=linkedin-in><i class="fab fa-linkedin-in"></i></a></li><li><a href="https://web.whatsapp.com/send?text=Clinical+Value+of+RNA+Sequencing-Based+Classifiers+for+Prediction+of+the+Five+Conventional+Breast+Cancer+Biomarkers%3A+A+Report+From+the+Population-Based+Multicenter+Sweden+Cancerome+Analysis+Network%E2%80%93Breast+Initiative%20https%3A%2F%2Fwww.brueffer.io%2Fpublication%2F2018_scan-b_biomarker_prediction%2F" target=_blank rel=noopener class=share-btn-whatsapp aria-label=whatsapp><i class="fab fa-whatsapp"></i></a></li><li><a href="https://service.weibo.com/share/share.php?url=https%3A%2F%2Fwww.brueffer.io%2Fpublication%2F2018_scan-b_biomarker_prediction%2F&title=Clinical+Value+of+RNA+Sequencing-Based+Classifiers+for+Prediction+of+the+Five+Conventional+Breast+Cancer+Biomarkers%3A+A+Report+From+the+Population-Based+Multicenter+Sweden+Cancerome+Analysis+Network%E2%80%93Breast+Initiative" target=_blank rel=noopener class=share-btn-weibo aria-label=weibo><i class="fab fa-weibo"></i></a></li></ul></div><div class="media author-card content-widget-hr"><a href=https://www.brueffer.io><img class="avatar mr-3 avatar-circle" src=/authors/cbrueffer/avatar_hu17f80c4ec327e7baf25f17a8159016b3_537581_270x270_fill_lanczos_center_3.png alt="Christian Brueffer"></a><div class=media-body><h5 class=card-title><a href=https://www.brueffer.io>Christian Brueffer</a></h5><h6 class=card-subtitle>Bioinformatician and Data Scientist</h6><p class=card-text>Freelance Bioinformatician and Data Scientist with interests including disease biology and diagnostics, particularly in cancer, and open source bioinformatics.</p><ul class=network-icon aria-hidden=true><li><a href=mailto:christian@brueffer.io><i class="fas fa-envelope"></i></a></li><li><a href=https://twitter.com/cbrueffer target=_blank rel=noopener><i class="fab fa-twitter"></i></a></li><li><a href=https://mastodon.social/@cbrueffer target=_blank rel=noopener><i class="fab fa-mastodon"></i></a></li><li><a href=https://www.linkedin.com/in/cbrueffer target=_blank rel=noopener><i class="fab fa-linkedin"></i></a></li><li><a href="https://scholar.google.com/citations?user=BFnR7W8AAAAJ" target=_blank rel=noopener><i class="ai ai-google-scholar"></i></a></li><li><a href=https://www.researchgate.net/profile/Christian_Brueffer2 target=_blank rel=noopener><i class="ai ai-researchgate"></i></a></li><li><a href=https://orcid.org/0000-0002-3826-0989 target=_blank rel=noopener><i class="ai ai-orcid"></i></a></li><li><a href=https://github.com/cbrueffer target=_blank rel=noopener><i class="fab fa-github"></i></a></li></ul></div></div></div></div></div><div class=page-footer><div class=container><footer class=site-footer><p class="powered-by copyright-license-text">© 2023 Christian Brueffer. This work is licensed under <a href=https://creativecommons.org/licenses/by-nc-nd/4.0 rel="noopener noreferrer" target=_blank>CC BY NC ND 4.0</a></p><p class="powered-by footer-license-icons"><a href=https://creativecommons.org/licenses/by-nc-nd/4.0 rel="noopener noreferrer" target=_blank aria-label="Creative Commons"><i class="fab fa-creative-commons fa-2x" aria-hidden=true></i>
<i class="fab fa-creative-commons-by fa-2x" aria-hidden=true></i>
<i class="fab fa-creative-commons-nc fa-2x" aria-hidden=true></i>
<i class="fab fa-creative-commons-nd fa-2x" aria-hidden=true></i></a></p><p class=powered-by>Published with <a href="https://wowchemy.com/?utm_campaign=poweredby" target=_blank rel=noopener>Wowchemy</a> — the free, <a href=https://github.com/wowchemy/wowchemy-hugo-themes target=_blank rel=noopener>open source</a> website builder that empowers creators.</p></footer></div></div><script src=/js/vendor-bundle.min.46271ef31da3f018e9cd1b59300aa265.js></script>
<script src=https://cdn.jsdelivr.net/npm/leaflet@1.7.1/dist/leaflet.min.js integrity crossorigin=anonymous></script>
<script id=search-hit-fuse-template type=text/x-template>
    <div class="search-hit" id="summary-{{key}}">
      <div class="search-hit-content">
        <div class="search-hit-name">
          <a href="{{relpermalink}}">{{title}}</a>
          <div class="article-metadata search-hit-type">{{type}}</div>
          <p class="search-hit-description">{{snippet}}</p>
        </div>
      </div>
    </div>
  </script><script src=https://cdn.jsdelivr.net/gh/krisk/Fuse@v3.2.1/dist/fuse.min.js integrity="sha512-o38bmzBGX+hD3JHWUFCDA09btWaqrNmoJ3RXLlrysA7PP01Kgs4UlE4MhelE1v5dJR3+cxlR4qQlotsW7jKsnw==" crossorigin=anonymous></script>
<script src=https://cdn.jsdelivr.net/gh/julmot/mark.js@8.11.1/dist/jquery.mark.min.js integrity="sha512-mhbv5DqBMgrWL+32MmsDOt/OAvqr/cHimk6B8y/bx/xS88MVkYGPiVv2ixKVrkywF2qHplNRUvFsAHUdxZ3Krg==" crossorigin=anonymous></script>
<script id=page-data type=application/json>{"use_headroom":true}</script><script src=/js/wowchemy-headroom.db4755770454eb63685f8de785c0a172.js type=module></script>
<script src=/en/js/wowchemy.min.8621d706896ab6d11dd8fd7042e20931.js></script>
<script src=/js/wowchemy-map.a26e9d2f7238ba5b868384f1c5bc6477.js type=module></script><div id=modal class="modal fade" role=dialog><div class=modal-dialog><div class=modal-content><div class=modal-header><h5 class=modal-title>Cite</h5><button type=button class=close data-dismiss=modal aria-label=Close>
<span aria-hidden=true>&#215;</span></button></div><div class=modal-body><pre><code></code></pre></div><div class=modal-footer><a class="btn btn-outline-primary my-1 js-copy-cite" href=# target=_blank><i class="fas fa-copy"></i> Copy</a>
<a class="btn btn-outline-primary my-1 js-download-cite" href=# target=_blank><i class="fas fa-download"></i> Download</a><div id=modal-error></div></div></div></div></div><script src=/js/wowchemy-publication.68f8d7090562ca65fc6d3cb3f8f2d2cb.js type=module></script>
<script type=text/javascript src=https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js></script>
<script async src=https://badge.dimensions.ai/badge.js></script></body></html>