---
title: "Monitoring ctDNA dynamics in early breast cancer using a novel ultra-sensitive tumor-informed structural variant approach combining whole-genome sequencing and multiplex dPCR"
authors:
- Lao H. Saal
- Niklas Loman
- Yilun Chen
- Miguel Alcaide
- cbrueffer
- Sergii Gladchuk
- Hina Dalal
- Pei Meng
- Anthony M. George
- Sofia Birkeälv
- Hani Saghir
- Siker Kimbung
- Daniel Förnvik
- Lina Zander
- Gabriella Honeth
- Karen Howarth
- Åke Borg
- Anna Ehinger
- Martin Malmberg
- Lisa Rydén

date: 2024-05-29
doi: "10.1200/JCO.2024.42.16_suppl.568"

add_badge: true

# Schedule page publish date (NOT publication's date).
#publishDate: "2017-01-01T00:00:00Z"

# Publication type.
# Legend: 0 = Uncategorized; 1 = Conference paper; 2 = Journal article;
# 3 = Preprint / Working Paper; 4 = Report; 5 = Book; 6 = Book section;
# 7 = Thesis; 8 = Patent
publication_types: ["1"]

# Publication name and optional abbreviated publication name.
publication: "*Journal of Clinical Oncology, 2024. 42:16_suppl*"
#publication_short: ""

abstract: "**Background**<br />

Persistent circulating cell-free tumor DNA (ctDNA) detection during neoadjuvant treatment of early-breast cancer (EBC) indicates high-risk disease. Detection of ctDNA post-resection of EBC (molecular residual disease, MRD) indicates occult metastatic disease and impending disease relapse. For ctDNA to be integrated into EBC management, accessible and scalable diagnostic tools are required. Here, we apply a highly sensitive, personalized tumor-informed approach to ctDNA evaluation predicated on analyses of structural variants (SVs) using a novel digital PCR (dPCR) SV technology.<br />

**Methods**<br />

170 patients with EBC and eligible for neoadjuvant therapy (NAT) were recruited through the prospective SCAN-B study (NCT02306096) between Dec 2014 and Mar 2019 (25.8% TNBC, 47.1% HR+/HER2- and 24.1% HER2+). Interim results are presented for the first 46 consecutive patients (comprising 567 plasma samples) where minimum QC criteria were met (10% tumor content and 10x sequencing depth). Whole genome sequencing (WGS) was performed on tumor material and personalized multiplex dPCR assays designed tracking up to 8 SVs for use in ctDNA analyses. Plasma samples were collected at baseline, during NAT, pre- and post-surgery and at 6-monthly intervals during follow up. Clinical characteristics and recurrence outcomes were recorded.<br />

**Results**<br />

ctDNA was detected at one or more timepoints prior to surgery in 43/46 (93%) patients across all breast cancer subtypes, at a median variant allele frequency (VAF) of 0.14% (range 0.0002% - 27.6%). ctDNA levels remained detectable at the end of NAT in 24% patients (11/46); 5/11 (45%) ctDNA positive patients experienced disease relapse versus 1/35 (3%) ctDNA negative patients (P=0.002, Fisher’s exact test). At one or more post-operative timepoints, ctDNA was detected in 6/6 (100%) patients who experienced clinical recurrence, with lead times up to 52 months (median 11.8 months, range 3.5 to 52 months). In 40 patients without presentation of clinical recurrence to date, ctDNA was undetectable across 285/287 (99.3%) plasma timepoints collected post-surgery. Post-operative detection of ctDNA associated with poor overall survival (OS) compared to absence of ctDNA (log-rank P<0.0001).<br />

**Conclusions**<br />

In this interim analysis of an ongoing prospective study in patients with EBC receiving NAT, we analyzed plasma for ctDNA using a novel tumor-informed dPCR assay tracking patient-specific SVs. ctDNA detection post-neoadjuvant therapy, prior to surgery associated with high-risk of disease relapse. Postoperative ctDNA detection was observed in 100% patients with clinical recurrence, and associated with poorer OS and long lead-times. Our data demonstrate the feasibility of SVs as an MRD analyte and provide evidence for high levels of clinical sensitivity achievable with this approach in EBC.<br />

**Clinical trial information**<br />

<a href='https://clinicaltrials.gov/study/NCT02306096'>NCT02306096</a>"

# Summary. An optional shortened abstract.
#summary: Lorem ipsum dolor sit amet, consectetur adipiscing elit. Duis posuere tellus ac convallis placerat. Proin tincidunt magna sed ex sollicitudin condimentum.

tags:
- Cancer
- Breast Cancer
- Bioinformatics
- Whole Genome Sequencing
- Structural Variants
- Liquid Biopsy
- Digital PCR
- ctDNA
- Molecular Residual Disease
featured: false

links:
- name: "PDF"
  url: "https://ascopubs.org/doi/pdf/10.1200/JCO.2024.42.16_suppl.568"
  icon: "file-pdf"
#url_pdf: ""
#url_preprint: ""
#url_code: ""
#url_dataset: ""
#url_project: ""
#url_slides: ""
#url_video: ""
#url_poster: ""
#url_source: ""


# Featured image
# To use, add an image named `featured.jpg/png` to your page's folder. 
image:
  caption: ''
  focal_point: "Center"
  preview_only: false

# Associated Projects (optional).
#   Associate this publication with one or more of your projects.
#   Simply enter your project's folder or file name without extension.
#   E.g. `internal-project` references `content/project/internal-project/index.md`.
#   Otherwise, set `projects: []`.
projects: []

# Slides (optional).
#   Associate this publication with Markdown slides.
#   Simply enter your slide deck's filename without extension.
#   E.g. `slides: "example"` references `content/slides/example/index.md`.
#   Otherwise, set `slides: ""`.
# slides: example
---

<html>
  <style>
    section {
        background: white;
        color: black;
        border-radius: 1em;
        padding: 1em;
        left: 50% }
    #inner {
        display: inline-block;
        display: flex;
        align-items: center;
        justify-content: center }
  </style>
  <section>
    <div id="inner">
      <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>
        <span style="float:left";
          class="__dimensions_badge_embed__"
          data-doi="10.1200/JCO.2024.42.16_suppl.568"
          data-hide-zero-citations="true"
          data-legend="always">
        </span>
      <script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>
        <div style="float:right";
          data-link-target="_blank"
          data-badge-details="right"
          data-badge-type="medium-donut"
          data-doi="10.1200/JCO.2024.42.16_suppl.568"
          data-condensed="true"
          data-hide-no-mentions="true"
          class="altmetric-embed">
        </div>
    </div>
  </section>
